src/common/components/Aisles/Aisles.module.scss

```diff
@@ -0,0 +1,19 @@
+.aisles {
+  color: #000;
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  ul {
```

Comment:
You can also use BEM notation for the list and list items: `.aisles__list`, `.aisles__list-item`.

Personally, I recommend BEM notation where possible to avoid specificity issues with CSS. For example, `.aisles li` has higher priority than `.aisles__list-item`. If you need to apply a CSS override, it's much easier if your selectors have the same priority :)

Reply:
The console was shouting at me when I used a hypen in class names. Just put it aside for the moment.